Tasks/DownloadExternalBuildArtifacts/task.json

```diff
@@ -0,0 +1,112 @@
+{
```

Comment:
We should add this task into vsts-rm-extensions repo & make sure it gets bundled with External TFS Artifacts extension.